modules/swagger-codegen/src/main/resources/Java/libraries/jersey2/ApiClient.mustache

```diff
@@ -682,8 +682,9 @@ public class ApiClient {
 
     Entity<?> entity = serialize(body, formParams, contentType);
 
-    Response response;
+    Response response = null;
 
+    try {
     if ("GET".equals(method)) {
```

Comment:
@wing328 I think I already changed the file correctly. Now I see running the ./bin/java-all-petstore.sh